### DISCUSSION TITLE
Remote - powering up & VLC

### DISCUSSION THREAD

**Discussion by floormanager1** [GerryDazoo/Slinger#141] [Q&A] [answered]
Hi, I've been configuring my remote and thought I'd screwed it as I could not get it to power on, or so I thought. I'd started VLC, then clicked power on but nothing appeared.  Actually, the power on worked but VLC would not show the video stream.  I had to close down VLC and reopen it and tada! it played.  As you cannot operate the remote without a video connection to the server, I have to open VLC, open the remote, power up, close VLC and reopen.  It's OK now I know this but is a  bit clunky.  Am I missing something?

**Comment by GerryDazoo** [comment, marked as answer]
I don't think you're missing anything. I can't reproduce your scenerio with a Windows VLC here at home. Two things, can you try a different media player and also can you confirm you have VideoSource configured in you config.ini.

**Comment by floormanager1** [comment]
Hi Gerry, video source is configured.  I'll try a different player.  it's not an issue, just wondering whether it's me.  It's almost VLC times out if it has no signal after a few seconds.

**Comment by vhdsoccer** [comment]
I can confirm what floormanager1 says, I have the same issue.
When the box I access via Slingbox is powered off, the power button on the remote does not seem to work. However, when I turn off everything (VLC and Slinger) then come back, the power is on.

**Reply by GerryDazoo** [reply]
This doesn't sound like FM's issue.  Somehow the box is magically turning itself on?
Run the sling software and verify the power button works as expected. If it does maybe your box isn't using keycode 1 for power. Or you may have the wrong [REMOTE] "code", Set it and your VideoSource. Are you connected via HDMI?

**Comment by tryin100** [comment]
I have the same behavior as floormanager1.  I have 3 sling 500s.  I have video source and input set to same value in config.sys (i.e. 1).
I start VLC.  I start remote.  I tap power on.  Both my cable (DTV) box and Sling start, but VLC kicks me out of video stream.  I reopen video stream and everything works fine.
I have no idea what a solution is.  I just open VLC a second time and watch.

**Reply by pfuller88** [reply]
i think VLC stops a stream as soon as it picks up any change in the video source.   Maybe when powering on the cable boxes they send a few video checks that VLC acts on.

**Reply by GerryDazoo** [reply]
You are correct sir. What you need to do is turn on "loop 1" on VLC and it will automatically reconnect. To be exact it's the slingbox that is trying to resync the video stream and VLC can't handle it and the connection drops.

**Comment by GerryDazoo** [comment]
What you need to do is turn on "loop 1" on VLC and it will automatically reconnect. To be exact it's the slingbox that is trying to resync the video stream and VLC can't handle it and the connection drops.

**Reply by tryin100** [reply]
Thanks.
For those users who may not readily know how to turn 'loop 1' on.  On my windows PC version:
Open VLC.  Choose ‘View’ and click on ‘Advanced Controls’ (to make loop option visible at bottom of VLC window), choose the icon that looks like two looping arrows facing left/right, if it’s grey click on it TWICE (should then turn blue and have a little ‘1’ between the arrows)

**Comment by Toei79** [comment]
I want add something,  if you have some no old asus router,  capable to use instant guard app from asus,  you can watch remotely without open ports.

Instant guard let you connect  to your local network  remotely  to your network 

Its like vpn but using your network
